kubernetes/loculus/templates/lapis-deployment.yaml

```diff
@@ -48,14 +48,22 @@ spec:
             periodSeconds: 10
             failureThreshold: 5
             timeoutSeconds: 5
-          livenessProbe:
+          readinessProbe:
             httpGet:
               path: /sample/info
               port: 8080
             initialDelaySeconds: 30
             periodSeconds: 10
             failureThreshold: 2
             timeoutSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /actuator/health
+              port: 8080
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            failureThreshold: 
```

Comment:
Just to check - I think as this is not set the failureThreshold will default to 3 - do we want this?

Kubernetes will check the health endpoint every 10 seconds (periodSeconds: 10),
If it fails 3 times in a row, it will consider the container unhealthy and restart it

Reply:
Thanks for spotting – setting real values does seem good I'm not sure what happened ed here